README.md

```diff
@@ -14,6 +14,8 @@ The detailed information about a template structure is available in [Generamba W
 
 * [Surf MVP Coordinatable module](https://github.com/surfstudio/generamba-templates/tree/master/surf_mvp_coordinatable_module) - generates a new module of architecture **Surf MVP** with routing using Coordinators.
 
+* [Surf MVP Coordinatable Alert module](https://github.com/surfstudio/generamba-templates/tree/master/surf_mvp_coordinatable_alert) - generates a new alert module of architecture **Surf MVP** with routing using Coordinators.
```

Comment:
I think it doesn't depend on coordinators, so we can name this template like "Surf MVP Alert module"

Reply:
I think it depends on coordinators.
1. It have no Router
2. Presenter is realization of ModuleOutput, what is incorrect in normal MVP module.

Reply:
@s4x Oh, I see, okay